lib/much-rails/records/validate_destroy.rb

```diff
@@ -34,6 +38,10 @@ def destroy!(as: :base, validate: true)
       destroy(validate: validate) || _raise_record_not_destroyed
     end
 
+    def destroy_without_validation!
+      destroy_without_validation
+    end
```

Comment:
@nweathersbee this isn't correct. needs to be more like this so it will actually raise an exception (make sense?):
```suggestion
    def destroy_without_validation!(as: :base)
      destroy!(as: as, validate: false)
    end
```